src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/JurisdictionServiceImpl.java

```diff
@@ -64,7 +64,7 @@ public void callCcd(JurisdictionUserCreationRequest request, String userId) {
                 responseCode = response.status();
                 throwException(responseCode);
             } else {
-                log.info(successMessage);
+                log.info(SUCCESS_MESSAGE);
```

Comment:
Please remove the else part ..not required

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/444/commits/1ca60a5369b53bae237ab6c14a9b9df1c4e83cf7